code/modules/uplink/uplink_items.dm

```diff
@@ -1344,7 +1344,7 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 			but you never know. Contents are sorted to always be worth 50 TC."
 	item = /obj/structure/closet/crate
 	cost = 20
-	player_minimum = 25
+	//pplayer_minimum = 25
```

Comment:
don't comment out code

Reply:
oh shit hahaha i was dicking around with supersurplus crates earlier this shouldnt have gotten through